finality-aleph/src/party/mod.rs

```diff
@@ -293,6 +293,33 @@ where
 const SESSION_STATUS_CHECK_PERIOD_MS: u64 = 1000;
 const SESSION_RESTART_COOLDOWN_MS: u64 = 500;
 
+fn get_authorities_for_session<'a, B, C>(
+    runtime_api: sp_api::ApiRef<'a, C::Api>,
+    session_id: SessionId,
+    first_block: NumberFor<B>,
+) -> Vec<AuthorityId>
+where
+    B: Block,
+    C: sp_api::ProvideRuntimeApi<B>,
+    C::Api: aleph_primitives::AlephSessionApi<B>,
+{
+    if session_id == SessionId(0) {
+        runtime_api
+            .authorities(&BlockId::Number(<NumberFor<B>>::saturated_from(0u32)))
+            .unwrap()
```

Comment:
Should also be an `expect`, even if it's extremely unlikely to fail.

Reply:
Done